e2e/host/zonefile

```diff
@@ -10,35 +10,42 @@
+_redirect.referer.host.host.example.com.               IN TXT "to=https://referer-redirect.host.host.example.com;type=host;ref=true"
 
-fallback.host.host.example.com.                  IN A 172.20.10.2
-_redirect.fallback.host.host.example.com.        IN TXT "v=txtv0;to=https://{label30};type=host;code=302" 
+fallback.host.host.example.com.                        IN A 172.20.10.2
+_redirect.fallback.host.host.example.com.              IN TXT "v=txtv0;to=https://{label30};type=host;code=302" 
 
-sourcerecord.host.host.example.com.              IN A 172.20.10.2
-upstreamrecord.host.host.example.com.            IN A 172.20.10.2
-_redirect.sourcerecord.host.host.example.com.    IN TXT "v=txtv0;type=host;use=_redirect.upstreamrecord.host.host.example.com" 
-_redirect.upstreamrecord.host.host.example.com.  IN TXT "v=txtv0;to=https://upstream.host.host.example.com;type=host;code=302" 
+sourcerecord.host.host.example.com.                    IN A 172.20.10.2
+upstreamrecord.host.host.example.com.                  IN A 172.20.10.2
+_redirect.sourcerecord.host.host.example.com.          IN TXT "v=txtv0;type=host;use=_redirect.upstreamrecord.host.host.example.com" 
+_redirect.upstreamrecord.host.host.example.com.        IN TXT "v=txtv0;to=https://upstream.host.host.example.com;type=host;code=302" 
 
 
-srcwithto.host.host.example.com.              IN A 172.20.10.2
-uprecord.host.host.example.com.               IN A 172.20.10.2
-_redirect.srcwithto.host.host.example.com.    IN TXT "v=txtv0;to=https://wrongredirect.host.host.example.com;type=host;code=302;use=_redirect.upstreamrecord.host.host.example.com" 
-_redirect.uprecord.host.host.example.com.     IN TXT "v=txtv0;to=https://upstream.host.host.example.com;type=host;code=302" 
\ No newline at end of file
+srcwithto.host.host.example.com.                       IN A 172.20.10.2
+uprecord.host.host.example.com.                        IN A 172.20.10.2
+_redirect.srcwithto.host.host.example.com.             IN TXT "v=txtv0;to=https://wrongredirect.host.host.example.com;type=host;code=302;use=_redirect.upstreamrecord.host.host.example.com" 
+_redirect.uprecord.host.host.example.com.              IN TXT "v=txtv0;to=https://upstream.host.host.example.com;type=host;code=302" 
+
+chaining.path.path.example.com.                        IN A 172.20.10.2
+chained.host.path.example.com.                         IN A 172.20.10.2
+_redirect.chaining.path.path.example.com.              IN TXT "v=txtv0;type=path;to=https://fallback-to.path.path.example.com;code=302"
+_redirect.releases.chaining.path.path.example.com.     IN TXT "v=txtv0;type=path;to=http://chained.host.path.example.com"
+_redirect._.chained.host.path.example.com.             IN TXT "v=txtv0;type=host;to=https://chained-redirect.host.path.example.com;code=302"
+_redirect.chained.host.path.example.com.               IN TXT "v=txtv0;type=host;to=https://chained-redirect.host.path.example.com;code=302"
```

Comment:
Why the double record here? We wouldn't know, which one got hit in the test. So not sure, why we need the wildcard as well as the direct record.

Reply:
Leftovers from when I was trying to reproduce the bug. The wildcard should be removed and we need "_redirect.chained.host.path.example.com" because "_redirect.releases.chaining.path.path.example.com" is pointed to it.